### ISSUE TITLE
As a user, I want a warning if I'm missing required claims so I know I have to enter them before I can sign up to an exchange

### ISSUE THREAD

**Issue by Mitchell139** [dltxio/idem-mobile#415]
Exchanges: Digital Surge, GPIB, Coinstash
## AC1

- Given I do not have the required claims for the exchange
- When I am on the exchange signup screen,
- Then I will not be able to tap the "Sign up" button
- And I see the text "[Exchage name] requires your [Claim], [Claim], and [Claim] to be completed."

- Or I see the text "[Exchage name] requires your [Claim] to be completed."

Instead of having an alert message for required claims, each exchange page should have the requirements listed on the screen that is to be entered before the user can sign up.   Releates to #407 

Site.json specifies what is required "**requiredClaimTypes**"

```json
  {
    "id": 5,
    "name": "Digital Surge",
    "tagline": "Bitcoin & Crypto made simple",
    "description": "A uniquely easy way to buy, sell & store crypto in Australia. Enjoy safe & effortless trading today.",
    "website": "https://digitalsurge.com.au/",
    "logo": "https://raw.githubusercontent.com/dltxio/idem-mobile/main/data/digital_surge.png",
    "signup": "https://digitalsurge.com.au/api/registration/sign-up/",
    "useProxy": true,
    "requiredClaimTypes": [
      { "EmailCredential": "required", "verified": true },
      { "NameCredential": "required", "verified": false },
      { "MobileCredential": "required", "verified": true }
    ],
    "backgroundColor": "#FFFFFF",
    "tempPassword": false,
    "passwordComplexity": ""
    "verifyClaims": false,
    "enable": true
  },
```

**Comment by bitcoinbrisbane** [comment]
@TomDodson notes

1. give a text box showing required claims
2. disable the button
3. remove the alert

**Comment by Mitchell139** [comment]
@bitcoinbrisbane what about this pop up when you navigate to the vendor details page? Uploading Screen_Recording_20220831-140529_idemID.mp4 …

**Comment by bitcoinbrisbane** [comment]
> * Given I am on an exchange screen and I do not have the required claims to signup to that exchange
> * Then I will not be able to tap the verify button
> 
> Add message below logo but above "Signup" button with list of claims that needs to be verified before the user can signup to that exchange
> 
> Site.json specifies what is required "**requiredClaimTypes**"
> 
> ```json
>   {
>     "id": 5,
>     "name": "Digital Surge",
>     "tagline": "Bitcoin & Crypto made simple",
>     "description": "A uniquely easy way to buy, sell & store crypto in Australia. Enjoy safe & effortless trading today.",
>     "website": "https://digitalsurge.com.au/",
>     "logo": "https://raw.githubusercontent.com/dltxio/idem-mobile/main/data/digital_surge.png",
>     "signup": "https://digitalsurge.com.au/api/registration/sign-up/",
>     "useProxy": true,
>     "requiredClaimTypes": [
>       { "EmailCredential": "required", "verified": true },
>       { "NameCredential": "required", "verified": false },
>       { "MobileCredential": "required", "verified": true }
>     ],
>     "backgroundColor": "#FFFFFF",
>     "tempPassword": false,
>     "passwordComplexity": ""
>     "verifyClaims": false,
>     "enable": true
>   },
> ```
> 
> Relates to #415
> 
> ![Untitled-2022-09-03-1121-2](https://user-images.githubusercontent.com/8411406/188251432-7d7a6edf-64e5-43ef-b9fc-ac537eadaa49.png)

@Mitchell139 needs a when.

**Comment by Mitchell139** [comment]
Doesn't take into account when exchanges require claims to be verified

**Comment by pondpiu** [comment]
[sites.json](https://github.com/dltxio/idem-mobile/blob/development/data/sites.json) don't have `verified` field yet.
can we have verified logic in another ticket? otherwise this ticket would be blocked by `https://github.com/dltxio/idem-mobile/issues/407`

**Comment by Mitchell139** [comment]
Can we make the claim names nice 
![image](https://user-images.githubusercontent.com/50130291/191898162-fb9204de-c4cb-40ff-9248-717f011761e2.png)



Filled out all claims but didn't verify (email was verified)
Went to exchanges page and then GPIB
Sign up button was enabled
Clicked sign up, got following error message
![image](https://user-images.githubusercontent.com/50130291/191899405-a42fa65f-74a9-4b13-a864-94d3785267da.png)
